gradoop-examples/src/main/java/org/gradoop/utils/sampling/statistics/functions/GetPropertyValueDouble.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.utils.sampling.statistics.functions;
+
+import org.apache.flink.api.common.functions.MapFunction;
+import org.gradoop.common.model.impl.pojo.GraphHead;
+
+/**
+ * Returns a property value as double
+ */
+public class GetPropertyValueDouble implements MapFunction<GraphHead, Double> {
```

Comment:
This class is used on one map only. You can replace it there with a lamda:
`.map(gh -> gh.getPropertyValue(PROPERTY_KEY_DENSITY).getDouble());`
It's possible that you have to call `.returns(new TypeHint<Double>() {});` after the map to tell whats the return of the map.